AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+# Test Case Title : Check that the basic client connect test works
+
+
+# fmt: off
+class Tests():
```

Comment:
nit: class Tests seems incorrectly named, "Tests" is generic and it doesn't contain tests but constant strings. Consider TestConstants or similar